nomad-{{cookiecutter.plugin_name}}/py_sources/tests/apps/test_app.py

```diff
@@ -2,3 +2,5 @@ def test_importing_app():
     # this will raise an exception if pydantic model validation fails for th app
     from nomad_{{cookiecutter.module_name}}.apps import myapp
 
+    assert myapp.app.label == 'MyApp'
```

Comment:
@hampusnasstrom I added an assertion here so the unused lint error shouldn't be there anymore.